### DISCUSSION TITLE
Increase usable ram for Manager

### DISCUSSION THREAD

**Discussion by wolfmatt21** [Security-Onion-Solutions/securityonion#4952] [Unsupported Versions] [answered]
Hi,

In the old security onion, you could increase the ram in the .jvm files but this has now gone. Is there anyway to allow the system to use more Ram as i have 64Gb useable but only using 28GB. Kibana is very slow to non-existent so need to find a fix to make it usable again.

**Comment by Git-Me-Some-Hub** [comment]
Does the Minion file have what you are looking for? /opt/so/saltstack/local/pillar/minions/<node>

**Comment by LS3-Allen** [comment]
I am having the same issue and just found this section in the documentation.

> We configure Redis to use 812MB of your total system memory. If you have sufficient RAM available, you may want to increase the redis_maxmemory setting in /opt/so/saltstack/local/pillar/global.sls. This value is in Megabytes so to set it to use 8 gigs of ram you would set the value to 8192.
> 
> Logstash on the manager node is configured to send to Redis. For best performance, you may want to ensure that batch is set to true and then tune the ls_pipeline_batch_size variable to find the sweet spot for your deployment.

**Reply by LS3-Allen** [reply]
I'm specifically having trouble with Redis, but there may be something in global.sls for Kibana also.

**Comment by wolfmatt21** [comment]
Thanks for the reply, does anyone know how to up the logstash ram as i have tried that minion file but thats doesnt make any difference.

**Comment by TOoSmOotH** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/search.html?q=heap&check_keywords=yes&area=default
